waspc/src/Wasp/Generator/DbGenerator/Common.hs

```diff
@@ -93,21 +90,8 @@ dbSchemaChecksumOnLastGenerateFileInDbRootDir = [relfile|schema.prisma.wasp-gene
 dbSchemaChecksumOnLastGenerateFileProjectRootDir :: Path' (Rel ProjectRootDir) (File DbSchemaChecksumOnLastGenerateFile)
 dbSchemaChecksumOnLastGenerateFileProjectRootDir = dbRootDirInProjectRootDir </> dbSchemaChecksumOnLastGenerateFileInDbRootDir
 
-prismaClientOutputDirEnvVar :: String
-prismaClientOutputDirEnvVar = "PRISMA_CLIENT_OUTPUT_DIR"
-
-prismaClientOutputDirInAppComponentDir :: AppComponentRootDir d => Path' (Rel d) (Dir ServerRootDir)
-prismaClientOutputDirInAppComponentDir = [reldir|node_modules/.prisma/client|]
-
-serverPrismaClientOutputDirEnv :: (String, String)
-serverPrismaClientOutputDirEnv = appComponentPrismaClientOutputDirEnv serverRootDirFromDbRootDir
-
-webAppPrismaClientOutputDirEnv :: (String, String)
-webAppPrismaClientOutputDirEnv = appComponentPrismaClientOutputDirEnv webAppRootDirFromDbRootDir
-
-appComponentPrismaClientOutputDirEnv :: AppComponentRootDir d => Path' (Rel DbRootDir) (Dir d) -> (String, String)
-appComponentPrismaClientOutputDirEnv appComponentDirFromDbRootDir =
-  (prismaClientOutputDirEnvVar, SP.fromRelDir $ appComponentDirFromDbRootDir </> prismaClientOutputDirInAppComponentDir)
+waspProjectDirFromProjectRootDir :: Path' (Rel ProjectRootDir) (Dir WaspProjectDir)
```

Comment:
Oh and as you said this should indeed go somewhere else, something like Generator/Common.hs , or even higher maybe.